### DISCUSSION TITLE
'功能'按钮的存在是否有些意义不明了

### DISCUSSION THREAD

**Discussion by caikun233** [the1812/Bilibili-Evolved#4894] [提问 / Q&A]
装好之后左侧会有这么个按钮
![图片](https://github.com/user-attachments/assets/121dc24e-b9c5-4596-b024-aecc8745d5e7)
点开后让我去文档里看怎么添加功能
![图片](https://github.com/user-attachments/assets/dd9458d4-b0e7-435a-b5c9-ed67dbac857c)
看了半天愣是没看到这个功能按钮有什么意义，功能不都是组件和插件提供的来着，对它们的调整和信息都在设置界面里啊（
并且我装了一些组件后，那个按钮里的东西也没变化

**Comment by Geeyun-JY3** [comment]
你再装多一些组件就知道这有什么意义了，`下载视频`、`下载字幕`、`查看封面`等功能的入口都在这里面。

#4731 刚好有些截图。

**Comment by the1812** [comment]
需要手动触发操作的功能, 在 b 站页面中也没有合适的位置放时, 就会放在这里
其他的还有一些可能会经常调整的设置也可以在功能里调
